src/sqlabstraction/implementations/query_select_sqlite.php

```diff
@@ -171,13 +171,13 @@ private function buildRightJoins()
             $reversedConditions = array_reverse( $rJoinPart['conditions'] );
 
             // adding first table.
-            list( $key, $val ) = each( $reversedTables ); 
+            $val = current( $reversedTables );
             $oneItemResult .= $val;
 
-            while ( list( $key, $nextCondition ) = each( $reversedConditions ) )
+            foreach( $reversedConditions as $key => $nextCondition )
```

Comment:
Please add a space after the ``foreach`` keyword.

Did you also run the tests?

Reply:
Done, tests work for me.